internal/alertmanager/currentalert/process.go

```diff
@@ -93,17 +95,15 @@ func (e *examiner) processAlert(ds disableScaling) error {
 		}
 	case "downScale":
 		if ds.Down {
-			e.Log.Info("downscale is skipped due to minimum broker count")
-			return nil
+			return errors.New("downscaling is skipped due to downscale limit")
```

Comment:
why do we return with error instead of just simply logging the fact the downscale is skipped? Not performing downscale when it's disabled it's not an error.

Reply:
If we just log the skipping, the alert will be marked as processed because of the handler checks error returned by `examineAlert`.
https://github.com/banzaicloud/kafka-operator/blob/feature/min-max-broker/internal/alertmanager/currentalert/current_alerts.go#L129-L133
In that case, we log the skipping as info, the `examineAlert` should return the skipping state as well.